xarray/core/common.py

```diff
@@ -490,6 +490,53 @@ def assign_coords(self, coords=None, **coords_kwargs):
 
         >>> _ = da.assign_coords({"lon_2": ("lon", lon_2)})
 
+        Note the same method applies to `Dataset` objects.
+
+        Convert `Dataset` longitude coordinates from 0-359 to -180-179:
+
+        >>> np.random.seed(0)
+        >>> ds = xr.Dataset(
+        ...     data_vars=dict(
+        ...         temperature=(["x", "y", "time"], 15 + 8 * np.random.randn(2, 2, 3)),
```

Comment:
We've generally tried to not use `np.random`, both because of the seed issue, and because it can be difficult for people to track the numbers through the example. What do you think about replacing it with `np.arange(12).reshape(2,2,3)`? Or something similar.

Reply:
Thanks for the suggestions @max-sixty. I borrowed my example from the top-level `xarray.Dataset` examples, but your idea makes good sense. I'm quite new to `xarray` and wanted to keep my example similar to others already in the docs. Happy to submit another change if you think that will improve things.

Also thanks for pointing our https://github.com/max-sixty/pytest-accept. I'll take a look!

Reply:
Yes, we've been gradually going through the examples and changing them, but doc PRs are rarer than code (and so very much appreciated!), and we still have quite a few left over on `random`.

If you're up for making the change that would be great. If you don't think you'll get to it, this is still a good improvement. Thanks @gregbehm !

Reply:
@max-sixty, I think working through the docs and adding examples will be a great way for me to learn `xarray`, so I'll put the change on my to-do list and try to get to it soon. 

One more noob question: how do we connect this PR with the issue that prompted it, #6336 so that can get closed?

Reply:
That would be great!

It used to be that saying "closes #6336" worked, but I've seen that not work at times. Often someone will just manually close it after merging...

Reply:
> It used to be that saying `closes #6336` worked

this *does* work, but only if you use exactly that spelling in markdown. Unfortunately, github displays urls to issues / PRs the same way, hence the confusion.

If it did work, "closes" will become some kind of help link (not sure what the official name is), and on the right you see a linked issue (section "development"). In general, I think you just have to make sure the issue is linked in that section for it to work.